From 9be89355b257212a3e557efbdc6ba022b66c729e Mon Sep 17 00:00:00 2001
From: Rohith Sharma K S <rohithsharmaks@apache.org>
Date: Thu, 27 Oct 2016 11:34:07 +0530
Subject: [PATCH 2151/2748] YARN-4555.
 TestDefaultContainerExecutor#testContainerLaunchError
 fails on non-english locale environment.
 Contributed by Takashi Ohnishi.

(cherry picked from commit b110c4b5e82d8310f13a22bba1c8afbcca80144f)

Change-Id: I00f0063fb0856dd6e210dae485ba5cc736510046
---
 .../nodemanager/TestDefaultContainerExecutor.java  |    1 +
 1 file changed, 1 insertion(+)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/TestDefaultContainerExecutor.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/TestDefaultContainerExecutor.java
index 3a6d372..fd3216d 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/TestDefaultContainerExecutor.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/TestDefaultContainerExecutor.java
@@ -270,6 +270,7 @@ public Object answer(InvocationOnMock invocationOnMock)
     ContainerId cId = mock(ContainerId.class);
     ContainerLaunchContext context = mock(ContainerLaunchContext.class);
     HashMap<String, String> env = new HashMap<String, String>();
+    env.put("LANG", "C");
 
     when(container.getContainerId()).thenReturn(cId);
     when(container.getLaunchContext()).thenReturn(context);
-- 
1.7.9.5

